### ISSUE TITLE
[BUG] Backupstore connection test fails on our CI

### ISSUE THREAD

**Issue by khushboo-rancher** [harvester/tests#829]
## What's the test to develop? Please describe

The test `test_backup_restore.py::TestBackupRestore::test_connection` is failing for both NFS and S3

```
AssertionError: Failed to update backup target to NFS with {'endpoint': 'nfs://10.84.102.1:/srv/nfsshare'}
E         API Status(400): {'kind': 'Status', 'apiVersion': 'v1', 'metadata': {}, 'status': 'Failure', 'message': 'admission webhook "validator.harvesterhci.io" denied the request: There is VMBackup in creating or deleting progress', 'reason': 'BadRequest', 'code': 400}
E       assert 200 == 400
E         +200
E         -400
```

**Comment by khushboo-rancher** [comment]
This is still failing sometime.

**Comment by khushboo-rancher** [comment]
It should be good if we fix this before 1.2.0 final rc, so that we know that backup restore is working fine.

**Comment by khushboo-rancher** [comment]
The fix is merged, let's watch the result on CI for few days and then close it.
